assets/src/domain/eventEditor/services/filterState/datetimes/useDatesListFilterStateManager.ts

```diff
@@ -55,7 +58,7 @@ const useDatesListFilterStateManager = (): FSM => {
 			setSales,
 			setStatus,
 		}),
-		[...Object.values(state), ...Object.values(entityFilterState.getState())]
+		[...datesFSValues, ...entityFSValues]
```

Comment:
@nicolad You were not happy with this, so I changed it 😄

Reply:
yes, thank you, that was too crowded there :D 
need to keep variable distancing these times